addon/mixins/data-adapter-mixin.js

```diff
@@ -86,6 +89,28 @@ export default Mixin.create({
   },
 
   /**
+    Adds request headers containing the authorization data as constructed
+    by the {{#crossLink "DataAdapterMixin/authorizer:property"}}{{/crossLink}}.
+
+    This method will only called in Ember Data 2.7 or greater, older versions
```

Comment:
`This method will only be called in Ember Data 2.7 or greater, older versions` ;)